airflow/decorators/base.py

```diff
@@ -457,6 +457,11 @@ def _expand(self, expand_input: ExpandInput, *, strict: bool) -> XComArg:
         end_date = timezone.convert_to_utc(partial_kwargs.pop("end_date", None))
         if partial_kwargs.get("pool") is None:
             partial_kwargs["pool"] = Pool.DEFAULT_POOL_NAME
+        if partial_kwargs.get("pool_slots") and partial_kwargs["pool_slots"] < 1:
```

Comment:
Why do you have to get `pool_slots` twice?

Reply:
I have pool_slots twice because I wanted to make sure pool_slots existed before comparing it to an integer in order to avoid a TypeError. I can make it less verbose in my next commit with `if partial_kwargs.get("pool_slots", 1) < 1:`. In this way I only call it once and if it doesn't exist, it returns 1 instead of None so a TypeError is avoided and the condition is also not met.

Reply:
what about...

```python
if "pool_slots" in partial_kwargs and partial_kwargs["pool_slots"] < 1:
```

Reply:
Or just do this in two lines?

Reply:
I can do that!

Reply:
I have split the if statement into two lines in my latest commit.